### PR TITLE
Add sensor type for VOC and NOc index

### PR DESCRIPTION
**Pull request by brentru**
Addresses https://github.com/adafruit/Wippersnapper_Components/pull/105

### CODE REVIEW

**Review by lorennorman — APPROVED** [review]

File: proto/wippersnapper/i2c/v1/i2c.proto [hunk 1]
```diff
@@ -183,6 +183,8 @@ enum SensorType {
   SENSOR_TYPE_UNITLESS_PERCENT               = 30; /** Percentage, unit-less, "{value}%". */
   SENSOR_TYPE_AMBIENT_TEMPERATURE_FAHRENHEIT = 31; /** Temperature of the air around a sensor, in degrees Fahrenheit, "{value}°F". */
   SENSOR_TYPE_OBJECT_TEMPERATURE_FAHRENHEIT  = 32; /** Temperature of the object a sensor is touching/pointed at, in degrees Fahrenheit, "{value}°F".*/
+  SENSOR_TYPE_VOC_INDEX                      = 33; /** Values are an index from 1-500 with 100 being normal, "${$v} VOC".*/
+  SENSOR_TYPE_NOC_INDEX                      = 34; /** Values are an index from 1-500 with 100 being normal, "${$v} NOx".*/
```
Comment (lorennorman) [inline proto/wippersnapper/i2c/v1/i2c.proto:187]: oops should be NOX, not NOC

**Review by lorennorman — CHANGES_REQUESTED** [review]
oops wait, missed something

**Comment by lorennorman** [comment]
Also, this is backwards from the Arduino side:
https://github.com/adafruit/Adafruit_Wippersnapper_Arduino/pull/386/files#diff-4fd1699337680fc989b7bbabfda2bcce692862572e0cbed691292563d2be298fR58-R59

File: proto/wippersnapper/i2c/v1/i2c.proto (hunk 1 above)
Comment (brentru) [reply proto/wippersnapper/i2c/v1/i2c.proto]: Fixed

**Comment by brentru** [comment]
@lorennorman  re arduin-side, see comment here: https://github.com/adafruit/Adafruit_Wippersnapper_Arduino/pull/386#discussion_r1059124136

**Pull request merged**